pkg/cmd/pkg/package_install.go

```diff
@@ -1,26 +1,160 @@
 package pkg
 
 import (
+	"crypto/sha256"
+	"encoding/json"
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/dcos/client-go/dcos"
 	"github.com/dcos/dcos-cli/api"
+	"github.com/dcos/dcos-cli/pkg/plugin"
+	"github.com/dcos/dcos-cli/pkg/prompt"
+	"github.com/spf13/afero"
 	"github.com/spf13/cobra"
+
+	"github.com/dcos/dcos-core-cli/pkg/cosmos"
+	"github.com/dcos/dcos-core-cli/pkg/pluginutil"
 )
 
 func newCmdPackageInstall(ctx api.Context) *cobra.Command {
-	var appOnly, cliOnly, global, yes bool
+	var appOnly, cliOnly, yes bool
 	var appID, optionsPath, version string
 
 	cmd := &cobra.Command{
 		Use:   "install <package-name>",
 		Short: "Install a package",
 		Args:  cobra.ExactArgs(1),
 		RunE: func(cmd *cobra.Command, args []string) error {
-			return invokePythonCLI(ctx)
+			packageName := args[0]
+
+			if cliOnly == appOnly {
+				cliOnly = true
+				appOnly = true
+			}
+
+			if optionsPath != "" {
+				_, err := os.Stat(optionsPath)
+				if err != nil {
+					return err
+				}
+			}
+
+			c, err := cosmos.NewClient(ctx, pluginutil.HTTPClient(""))
+			if err != nil {
+				return err
+			}
+
+			description, err := c.PackageDescribe(packageName, version)
+			if err != nil {
+				return err
+			}
+			pkg := description.Package
+
+			link := "https://mesosphere.com/catalog-terms-conditions/#certified-services"
+			if !pkg.Selected {
+				fmt.Fprint(ctx.Out(), "This is a Community service. "+
```

Comment:
These informational messages should go to stderr according to https://github.com/dcos/dcos-cli/blob/master/design/style.md#stdoutstderr, I think this refactoring is a good time to switch from stdout to stderr.

Reply:
Fixed